src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -2024,6 +2030,49 @@ private bool VarIdentifierEndsWithSuffix(ParseNode node)
                  prevChild.Token.Type == TokenType.suffixterm);
         }
```

Comment:
I sometimes hate the way git's diff shows things.  This ends up looking like a much more complex change until I realized it's mostly moving the clauses in the if/else ladder to a new order, which git diff doesn't quite "get" so it shows it as a lot of individual edits instead of just moved code.